src/shared/components/SideNavbar/FollowedChannels.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react'
+import {
+  ChannelsWrapper,
+  ChannelsTitle,
+  ChannelsList,
+  ChannelsItem,
+  ShowMoreButton,
+  StyledChannelLink,
+} from './FollowedChannels.style'
+import { transitions } from '@/shared/theme'
+import { CSSTransition } from 'react-transition-group'
+import Icon from '../Icon'
+import { FollowedChannel } from '@/hooks/usePersonalData/localStorageClient/types'
+
+const MAX_CHANNELS = 4
+
+type FollowedChannels = {
+  followedChannels: FollowedChannel[]
+  expanded: boolean
+  onClick: () => void
+}
+
+const FollowedChannels: React.FC<FollowedChannels> = ({ followedChannels, expanded, onClick }) => {
+  const [isShowingMore, setIsShowingMore] = useState(false)
+
+  const numberOfChannels = followedChannels.length
+  const channelsToSlice = isShowingMore ? numberOfChannels : MAX_CHANNELS
+  const channels = numberOfChannels > MAX_CHANNELS ? followedChannels.slice(0, channelsToSlice) : followedChannels
```

Comment:
I don't think we need that condition here

```suggestion
  const channels = followedChannels.slice(0, channelsToSlice)
```